app/models/user.rb

```diff
@@ -25,7 +25,8 @@ def contributions_count
   end
 
   def password_complexity
-    return if password =~ /^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/
+    return if provider != 'email' ||
+              password =~ /^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/
```

Comment:
If you are going to fix the regex to allow symbols in a separate pull request, create an issue for it so you don't forget